packages/core/src/Button.js

```diff
@@ -65,6 +65,17 @@ const variations = {
         box-shadow: inset 0 0 0 2px ${getPaletteColor('dark')(props)};
       `}
     }
+  `,
+  link: css`
+    color: ${getPaletteColor('base')};
+    line-height: ${props => props.theme.lineHeights.standard};
```

Comment:
i added this only for this variation because `line-height` for `Button` was set to 1.5. I wonder if we can set the `Button` line-height to 1.4 which would affect all variations.

Reply:
@wesleymartin85 what do you think about Auggie's suggestion of changing the `line-height` on `Button`?

Reply:
@sdalonzo @abautist this could be helpful for when text as a button is inline with copy. But I know there are more use cases to think about. I started to look into this for all (most) scenarios. I think the next step is to review in a larger effort with a few designers and some devs to make sure we are all aligned and educated on the differences in buttons and links.

Reply:
Ok cool, I've kept the line height change as is so as not to impact other Button variations for now and we can discuss as a group for long-term alignment on line-height for buttons.